external/CMakeLists.txt

```diff
@@ -147,15 +153,15 @@ set(CPP_RTTI_ENABLED ON)
 set(ENABLE_ALL_WARNINGS OFF)
 
 # TODO: These options enable vectorization optimizations
-set(USE_SSE4_1 ON)
-set(USE_SSE4_2 ON)
-set(USE_AVX ON)
-set(USE_AVX2 ON)
+set(USE_SSE4_1 OFF)
+set(USE_SSE4_2 OFF)
```

Comment:
we can keep this on but keep in mind that those were recommended from the jolt implementation from their example. If things break in jolt we should remember this place to come back later

Reply:
Wait I will check if I accidentally changed this

Reply:
My mistake, fixed it

Reply:
Yup, AVX512 was the only one disabled (seems like its a new flag and some processors might not support it)